posts/2022-09-19-app-server-crafters-makers.adoc

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "Open Liberty- A brief history in maker projects of a cloud-native Java runtime
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/lauracowen
+author_github: https://github.com/lauracowen
+seo-title: TITLE - OpenLiberty.io
+seo-description: A look back at some of the interesting and fun projects that crafters and makers on the Liberty team have developed over its 10 year history
+blog_description: "A look back at some of the interesting and fun projects that crafters and makers on the Liberty team have developed over its 10 year history"
+open-graph-image: https://openliberty.io/img/blog/crafters-rug-mug-quilt.jpg
+---
+= Open Liberty: A brief history in maker projects of a cloud-native Java runtime
+Laura Cowen <https://github.com/lauracowen>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+:figure-caption!:
+//Blank line here is necessary before starting the body of the post.
+
+Five years ago today, in 2017, link:https://openliberty.io/blog/2017/09/19/open-sourcing-liberty.html[IBM published the source code of Liberty], its Java cloud-native runtime, on GitHub as link:https://github.com/openliberty/open-liberty[the Open Liberty project]. Liberty was actually born five years earlier, in 2012 (recently celebrating its 10th birthday), to address the need for a small footprint, extensible, and easily configurable Java runtime that could run on the cloud as well as in a data centre. Liberty was designed to provide link:https://developer.ibm.com/articles/why-cloud-native-java-developers-love-liberty/[a friendly experience to developers] as well as to be powerful and flexible at runtime.
```

Comment:
```suggestion
Five years ago, in 2017, link:https://openliberty.io/blog/2017/09/19/open-sourcing-liberty.html[IBM published the source code of Liberty], its Java cloud-native runtime, on GitHub as link:https://github.com/openliberty/open-liberty[the Open Liberty project]. Liberty was actually born five years earlier, in 2012 (recently celebrating its 10th birthday), to address the need for a small footprint, extensible, and easily configurable Java runtime that could run on the cloud as well as in a data centre. Liberty was designed to provide link:https://developer.ibm.com/articles/why-cloud-native-java-developers-love-liberty/[a friendly experience to developers] as well as to be powerful and flexible at runtime.
```

Reply:
Please can you also change the publishing date as appropriate and change the slug to `history-maker-projects` before publishing?